src/editor.c

```diff
@@ -20,15 +20,22 @@ void enable_raw_mode(e_context* ctx) {
 }
 
 
+/* sigh */
+void write_wrapped(int file, const char* str, int len) {
+   ssize_t x = write(file, str, len);
+   (void) x; 
+}
+
```

Comment:
Disclaimer: I'm not a C programmer in what so ever way. ;-) 
Wouldn't be nice to check for return value of `write` and name the function following your naming convention of `e_foobar`?

```
void e_write(int file, const char* str, int len) {
    if (write(file, str, len) != len) {
        fprintf(stderr, "Cannot write to file %i with error: %s\n", file, strerror(errno));
    }
}
```

Reply:
I understand your concern here. A lot of the time in C we don’t do all of the error checking we could do because we assume that some calls never fail—in `e` I also don’t check for failures of `malloc`/`realloc` for instance. I’m also not so sure about `fprintf`ing the error here, especially after the last call to `write` failed and considering that the user could redirect stderr, leading to weird artifacts on the screen (in the best) case.

As per the naming convention: the `e_<function>` convention is only really important for exported functions (as in, functions that are in the header) I feel, but I would agree that it’s less of a code smell to follow it regardless. I shall update it in the next commit.